server/data/wikidata/queries/works_reverse_claims.coffee

```diff
@@ -5,26 +5,8 @@ module.exports =
     """
     SELECT DISTINCT ?item WHERE {
       ?item wdt:#{pid} wd:#{qid} .
-      # book
-      { ?item wdt:P31 wd:Q571 . }
-      # literary work
-      UNION { ?item wdt:P31 wd:Q7725634 . }
-      # comic book album
-      UNION { ?item wdt:P31 wd:Q2831984 . }
-      # comic book
-      UNION { ?item wdt:P31 wd:Q1004 . }
-      # manga
-      UNION { ?item wdt:P31 wd:Q8274 . }
-      # book series
-      UNION { ?item wdt:P31 wd:Q277759 . }
-      # comic book series
-      UNION { ?item wdt:P31 wd:Q14406742 . }
-      # manga series
-      UNION { ?item wdt:P31 wd:Q21198342 . }
-      # novel series
-      UNION { ?item wdt:P31 wd:Q1667921 . }
-      # Filter-out entities tagged as both work and edition
-      FILTER NOT EXISTS { ?item wdt:P31 wd:Q3331189 }
+      ?item wdt:P31 ?work_type .
+      VALUES (?work_type) { (wd:Q571) (wd:Q47461344) (wd:Q7725634) (wd:Q2831984) (wd:Q1004) (wd:Q1760610) (wd:Q8274) } .
```

Comment:
before ther was some comment on what Qids meant, which made it more readable

Reply:
done in befb87f8